homeassistant/components/camera/strings.json

```diff
@@ -34,5 +34,66 @@
         }
       }
     }
-  }
+  },
+  "services": {
+    "turn_off": {
+      "name": "Turn off",
+      "description": "Turns off the camera."
+    },
+    "turn_on": {
+      "name": "Turn on",
+      "description": "Turns on the camera."
+    },
+    "enable_motion_detection": {
+      "name": "Enable motion detection",
+      "description": "Enables the motion detection."
+    },
+    "disable_motion_detection": {
+      "name": "Disable motion detection",
+      "description": "Disables the motion detection."
+    },
+    "snapshot": {
+      "name": "Take snapshot",
+      "description": "Takes a snapshot from a camera.",
+      "fields": {
+        "filename": {
+          "name": "Filename",
+          "description": "Template of a filename. Variable available is `entity_id`."
+        }
+      }
+    },
+    "play_stream": {
+      "name": "Play stream",
+      "description": "Plays camera stream on a supported media player.",
+      "fields": {
+        "media_player": {
+          "name": "Media player",
+          "description": "Media players to stream to."
+        },
+        "format": {
+          "name": "Format",
+          "description": "Stream format supported by media player."
+        }
+      }
+    },
+    "record": {
+      "name": "Record",
+      "description": "Record live camera feed.",
+      "fields": {
+        "filename": {
+          "name": "[%key:component::camera::services::snapshot::fields::filename::name%]",
+          "description": "Template of a filename. Variable available is `entity_id`. Must be mp4."
+        },
+        "duration": {
+          "name": "Duration",
+          "description": "Targeted recording length."
```

Comment:
```suggestion
          "description": "Planned duration of the recording (in seconds). The actual duration may vary. Default: 30&nbsp;s."
```

Reply:
The default is already shown in the UI, as is the measurement unit; both are defined in the selector. Should we add that to the translation as a duplicate?

Reply:
No. I should have double-checked the UI.